INDEX.HTML

```diff
@@ -0,0 +1,6 @@
+<html>
+
+
+<body>Hello Microverse!</body>
+
+</html>
```

Comment:
- [ ] [REQUIRED] It looks like the `head` tag and its components are missing here. The Meta charset attribute specifies the character encoding for the HTML document. Kindly add in order to fix the `Lighthouse` errors currently on the linters.

Reply:
I did that, however the result is still an error

Reply:
Hi @amklass11, below is the report of the stylelint error. Try to fix them all and you should be fine.
![Screenshot from 2022-02-16 08-50-36](https://user-images.githubusercontent.com/58692590/154219939-75a80a97-82ec-4ea1-84ca-ab5810177269.png)
If you also want to see the errors, please click on details right in front of the linters on this PR.

Reply:
okay thank you what about the light house error

Reply:
Hello, My apologies meta charset was included however error still persist